lib/publiccloud/ssh_interactive.pm

```diff
@@ -88,7 +90,7 @@ sub ssh_interactive_leave {
         send_key 'ctrl-c';
         send_key 'ctrl-c';
         send_key 'ret';
-        last if (script_run("true", timeout => 5, die_on_timeout => 0) == 0);
+        last if (&$test);
```

Comment:
Why do you test here `&` instead of simply the $test content?

Reply:
its function dereference so its equivalent to `$test->()` only three characters shorter

Reply:
is +- explicit , but too old syntax, and probably not best possible , so changing into arrow dereference